settings.gradle

```diff
@@ -161,19 +161,21 @@ project(":beam-sdks-java-io-synthetic").dir = file("sdks/java/io/synthetic")
 include "beam-sdks-java-javadoc"
 project(":beam-sdks-java-javadoc").dir = file("sdks/java/javadoc")
 include "beam-sdks-java-load-tests"
-project(":beam-sdks-java-load-tests").dir = file("sdks/java/load-tests")
+project(":beam-sdks-java-load-tests").dir = file("sdks/java/testing/load-tests")
 include "beam-sdks-java-maven-archetypes-examples"
 project(":beam-sdks-java-maven-archetypes-examples").dir = file("sdks/java/maven-archetypes/examples")
 include "beam-sdks-java-maven-archetypes-starter"
 project(":beam-sdks-java-maven-archetypes-starter").dir = file("sdks/java/maven-archetypes/starter")
 include "beam-sdks-java-nexmark"
-project(":beam-sdks-java-nexmark").dir = file("sdks/java/nexmark")
+project(":beam-sdks-java-nexmark").dir = file("sdks/java/testing/nexmark")
```

Comment:
Please update the website as well fixing links for the new paths:
```
./website/src/_includes/section-menu/sdks.html:25:    <li><a href="{{ site.baseurl }}/documentation/sdks/java/nexmark/">Nexmark benchmark suite</a></li>
./website/src/documentation/sdks/nexmark.md:5:permalink: /documentation/sdks/java/nexmark/
./website/src/documentation/sdks/nexmark.md:82:[NexmarkConfiguration.java](https://github.com/apache/beam/blob/master/sdks/java/nexmark/src/main/java/org/apache/beam/sdk/nexmark/NexmarkConfiguration.java)).
./website/src/documentation/sdks/nexmark.md:649:        sdks/java/nexmark/build/libs/beam-sdks-java-nexmark-{{ site.release_latest }}-spark.jar \
./website/src/documentation/sdks/java.md:48:- [Nexmark]({{site.baseurl}}/documentation/sdks/java/nexmark) is a benchmark suite that runs in batch and streaming modes.
```

Reply:
done